File: lua/test/exists_test.lua
-- SoftvpnMobile SDK exists test

local sdk = require("softvpn-mobile_sdk")

describe("SoftvpnMobileSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# SoftvpnMobile SDK exists test

require "minitest/autorun"
require_relative "../SoftvpnMobile_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = SoftvpnMobileSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
